server/command.go

```diff
@@ -239,6 +245,100 @@ If you see an option to create a Jira issue, you're all set! If not, refer to ou
 	return responsef(addResponseFormat, p.GetSiteURL(), ji.GetMattermostKey(), pkey)
 }
 
+// executeUninstallCloud will uninstall the jira cloud instance if the url matches, and then update all connected
+// clients so that their Jira-related menu options are removed.
+func executeUninstallCloud(p *Plugin, c *plugin.Context, header *model.CommandArgs, args ...string) *model.CommandResponse {
+	authorized, err := authorizedSysAdmin(p, header.UserId)
+	if err != nil {
+		return responsef("%v", err)
+	}
+	if !authorized {
+		return responsef("`/jira uninstall` can only be run by a system administrator.")
+	}
+	if len(args) != 1 {
+		return help()
+	}
+	jiraURL := args[0]
+
+	ji, err := p.LoadCurrentJIRAInstance()
+	if err != nil {
+		return responsef("No current Jira instance to uninstall")
+	}
+
+	jci, ok := ji.(*jiraCloudInstance)
+	if !ok {
+		return responsef("The current Jira instance is not a cloud instance")
+	}
+
+	if jiraURL != jci.GetURL() {
+		return responsef("You have entered an incorrect URL. The current Jira instance URL is: `" + jci.GetURL() + "`. Please enter the URL correctly to confirm the uninstall command.")
+	}
+
+	err = p.DeleteJiraInstance(jci.GetURL())
+	if err != nil {
+		return responsef("Failed to delete Jira instance " + ji.GetURL())
+	}
+
+	// Notify users we have uninstalled an instance
+	p.API.PublishWebSocketEvent(
+		wSEventInstanceStatus,
+		map[string]interface{}{
+			"instance_installed": false,
+		},
+		&model.WebsocketBroadcast{},
+	)
+
+	const uninstallInstructions = `Jira instance successfully disconnected. Go to XXX to remove the application in your Jira Cloud instance.`
```

Comment:
```suggestion
	const uninstallInstructions = `Jira instance successfully disconnected. Go to **Settings > Apps > Manage Apps** to remove the application in your Jira Cloud instance.`
```

Reply:
(Not blocking this PR) We would ideally make this a link to the appropriate page, similar to what we do in `/jira install` command output https://github.com/mattermost/mattermost-plugin-jira/blob/master/server/command.go#L209

Reply:
@jasonblais can we make that a quick ticket? I'll take it. I want to merge this. :)

Reply:
Yes, like I said, not blocking this PR :)

Reply:
https://mattermost.atlassian.net/browse/MM-15903